project.clj

```diff
@@ -23,6 +23,6 @@
             [lein-ring "0.12.0"]]
   :ring {:handler service.routing.core/app
          :auto-reload? true}
-  :jvm-opts ["-Xmx300m"])
+  :jvm-opts ["-Xmx500m"])
```

Comment:
@mehdisadeghi I dont think we would gain anything from that. In general I think we should make it as small as possible so I dont know if we would gain anything by defining the minimum size.